src/mongoose.ts

```diff
@@ -111,48 +112,35 @@ export function htMongooseFactory(mongoose: any) {
     };
   }
 
```

Comment:
i just realized all these Object.keys(...).includes(...) everywhere - it's pretty costly considering it's only there to protect a developer from shooting themselves.
I'd say change them all to
- [ ] `if(requestData[propertyKeyOfDocument]) {` everywhere